lib/helpers/rule-test-harness.js

```diff
@@ -243,6 +243,15 @@ module.exports = function generateRuleTests({
             );
           }
 
+          for (const act of actual) {
+            if (act.isFixable) {
+              assert(
+                item.fixedTemplate !== undefined,
+                'fixable test cases must assert the `fixedTemplate`'
```

Comment:
could we show "template" itself in error or somehow point to exact case to developer?

Reply:
@lifeart I added an example of the output to the PR description. The `assert` shows up under the specific test case, so I don't think we need to repeat it.